CDCgov/prime-reportstream
# Cle/6642 phone number

This PR updates the FHIR -> HL7 mapping to properly handle phone numbers.

Test Steps:
1. Run automated tests, verify they pass
OR
1. Use fhirdata command to generate an HL7 file from Fhir, verify the XTN values for the phone number are populated as expected. `./prime fhirdata -s metadata/hl7_mapping/ORU_R01/ORU_R01-base.yml -i <input fhir path> -o <output hl7 path>`

## Changes
- Added PhoneUtilities.kt
- Added PhoneUtilitiesTests.kt
- Updated telecom yml to include XTN 5, 6, 7, 8, 12
- Added tech debt notes (after talking to Mo) to Element.kt surrounding how we handle phone parsing
